documentation/book/ref-kafka-logging.adoc

```diff
@@ -38,7 +38,7 @@ Components and their loggers are listed below.
 ** `mirrormaker.root.logger`
 
 * Topic Operator
-** `rootLogger.level`
+** `property.entity-topic-operator.root.logger`
```

Comment:
I do not understand this change - why are you changing the logger name?

Reply:
Until this PR, this worked with with `rootLogger.level` ... so I'm confused why we need now something so long and complicated. It seems to have also completely different naming schema compared to all other loggers we have.

Reply:
If you take a look into created CM it is something like 
```
property.entity-topic-operator.root.logger=DEBUG
rootLogger.level=${entity-topic-operator.root.logger}
```
So I need to parse this "ugly" logger. It can be renamed of course.

Reply:
I think this is fine for the generated CM content it self. But we should keep the `rootLogger.level` as the user facing configuration.

Reply:
And for the record: this also makes backwards incompatible change.